nixos/modules/services/hardware/keyd.nix

```diff
@@ -0,0 +1,68 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  meta = {
+    maintainers = [ "cidkid" ];
+  };
+
+  options = {
+    services.keyd = {
+      enable = mkOption {
```

Comment:
Please use ` enable = mkEnableOption "keyd daemon for remapping keys";`

Reply:
fixed